src/foam/u2/view/FObjectArrayTableView.js

```diff
@@ -0,0 +1,37 @@
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'FObjectArrayTableView',
+  extends: 'foam.u2.View',
+  requires: [
+    'foam.comics.v2.DAOBrowserView',
+    'foam.dao.MDAO'
+  ],
+  properties: [
+    {
+      class: 'Class',
+      name: 'of',
+      expression: function(data) {
+        return data[0].cls_;
+      }
+    }
+  ],
+  methods: [
+    function fromProperty(p) {
+      this.of = p.of;
+    },
+    function initE() {
+      var self = this;
+      self.SUPER();
+      self.add(self.slot(function(data) {
+        var dao = self.MDAO.create({
+          of: self.of
+        });
+        data.forEach(d => dao.put(d));
```

Comment:
Why do you need to copy the data from 'data' to 'dao'? Can't you just use 'data' directly?

Reply:
I think the DAOBrowserView expects data to be of DAOProperty, and the data here is just a FObjectArray